pages/Backpacks.md

```diff
@@ -0,0 +1,15 @@
+Backpacks are items in which you can store other items.  
+These can be very useful when looting or scavenging.
+
+These come in 6 tiers, each tier providing more inventory slots to store items in.
+
+## Bacpack Tiers
+
+| Tier             | Inventory Slots |
+| :--------------- | :-------------: |
+| Small Bacpack    | 9               |
+| Backpack         | 18              |
+| Large Bacpack    | 27              |
+| Woven Backpack   | 36              |
+| Gilded Bacpack   | 45              |
+| Radiant Backpack | 54              |
```

Comment:
Could mention the Restored Backpack too just for completion (the one you get from /sf backpack to retrieve lost content, uncraftable etc.); it would not concern players so it would probably need to be separated from the others to distinguish it as it's a special case but I don't think it appears anywhere else on the wiki so this sounds like the best place to at least mention it